<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CropWidget</class>
 <widget class="QWidget" name="CropWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="LeftVerticalLayout">
       <item>
        <widget class="QGroupBox" name="imageCoreGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Image</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="PreviewVerticalLayout_2"/>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="RightVerticalLayout">
       <item>
        <widget class="QGroupBox" name="previewCoreGroupBox">
         <property name="title">
          <string>Preview</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="PreviewVerticalLayout">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="labelPreviewImage">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="previewCoreGroupBox_2">
         <property name="title">
          <string>Current Selection</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Width:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_width">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Height:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_height">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="1">
             <widget class="QLabel" name="label_x">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>X Pos:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Y Pos:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_y">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="verticalCoreGroupBox">
         <property name="title">
          <string>Selection Settings</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="OptionVerticalLayout">
          <item>
           <layout class="QHBoxLayout" name="shapeHorizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Shape:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboOutputShape">
              <item>
               <property name="text">
                <string>Rect/Square</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Circle/Ellipse</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Cropper:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboCropperShape">
              <item>
               <property name="text">
                <string>Rect</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Square</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Fixed Rect</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ellipse</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Circle</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Fixed Ellipse</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>FixedSize:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="editCropperFixedWidth"/>
            </item>
            <item>
             <widget class="QSpinBox" name="editCropperFixedHeight"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Opacity:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="coreSlider" name="sliderOpacity">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnQuit">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSavePreview">
           <property name="text">
            <string>Crop</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>coreSlider</class>
   <extends>QSlider</extends>
   <header>coreslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
